scan-project/sample/config/scan-application.yml

```diff
@@ -1,7 +1,7 @@
 read:
   # !!! Project folder to be scanned - provide actual value !!!
-  folder: "/work/project/scanned-sample"
-
+  #folder: "/work/project/scanned-sample"
+  folder: "C:/Work/projects/code-egress-sample-scan"
```

Comment:
Local change?

Reply:
I see it has been removed

Reply:
No, it is still here

Reply:
Ah I was looking at the other file. So now finally reverted